specs/python/update_pip.yml

```diff
@@ -1,9 +1,9 @@
 ---
 name: Update all pip3 packages
-command: "pip3 list --outdated --format=freeze | cut -d = -f 1 | xargs -n1 pip3 install -U"
+command: "pip3 install -U $(pip3 freeze | cut -d '=' -f 1)"
```

Comment:
Is there a `pip-tools` command here that we could run instead? The [URL](https://coderwall.com/p/quwaxa/update-all-installed-python-packages-with-pip) you linked explicitly says to not use this command as it is not safe